src/language-handlebars/utils.js

```diff
@@ -41,6 +41,7 @@ const voidTags = new Set(htmlVoidElements);
 function isVoid(node) {
   return (
     voidTags.has(node.tag) ||
```

Comment:
Should we remove the html void tag check? 

`<img>` should always self closed.  If anyone wrote `<img></img>`, we can just ignore.